assets/ZZZ/components/ui/styles/themes/default/index.js

```diff
@@ -1,8 +0,0 @@
-import './color-variables.css';
```

Comment:
was getting this error:

![image](https://user-images.githubusercontent.com/1477453/79741637-42b08e00-830a-11ea-9302-9f85ba6ac9c9.png)

Reply:
we have this in `EDTR/refactor-master` in case we might need this, no need to keep this here